### ISSUE TITLE
Not a Issue  but i have a question.

### ISSUE THREAD

**Issue by Therealjosephchrzempiec** [thedigicraft/livestreams#1]
Hello i was wondering if it is possible to store the 1 or 0 in mysql instead of a text file?  Can you help me out please? I'm not good at coding but hardware is where my skills are at.

Joseph

**Comment by Jaunty42** [comment]
Yes you can store 1 or 0.

**Comment by Jaunty42** [comment]
So let me know your requirement. I guess you want to store/update values from nodemcu to mysql in format 1 or 0 right ?

**Comment by Therealjosephchrzempiec** [comment]
Hello yes to store and update the 1 and 0 in mysql.

**Comment by Jaunty42** [comment]
//Okay so first you have to send request to host server from nodemcu

int code = 1; // This is the value that you want to store/update
       HTTPClient http;
      http.begin("http://yourdomain.com/index.php?updatecode="+code); //Specify the URL

       int httpCode = http.GET();    
       if (httpCode > 0) { //Check for the returning code
        String payload = http.getString();
        Serial.println(httpCode);
        client.println(payload+"\r");
      }
    else {
       Serial.println(httpCode);
      Serial.println("Error on HTTP request&quot");
    }
    http.end(); //Free the resources

**Comment by Jaunty42** [comment]
// server side in PHP
//First you have to create table in mysql and insert default entries, say id = 1, and code = 1
// I have assumed table in this code -> arduino_table
// table name  = "arduino_table" and  Table Column -->1.)  id (INT) Primary, AutoIncrement 2.) code (INT).

```
<?php

$servername = "localhost";  //host address
$username = "username";  // database username
$password = "password";  // database password
$database = "database";  // database name

// Create connection
$conn = mysqli_connect($servername, $username, $password,$database);

// Check connection
if (!$conn) {
    die("Connection failed: " . mysqli_connect_error());
}
//echo "Connected successfully";

$code = $_GET['updatecode'];

if(!empty($code)) {

if ($code == 0 || $code == 1) {

$sql	= "UPDATE arduino_table SET  code='$code' WHERE id=1"; 	

$conn->query($sql);

$conn->close();
}

}

?>
```

**Comment by Therealjosephchrzempiec** [comment]
Hello once again I think i screwed up. I figured out most parts including the php and most of the arduino code but i get a error when uploading the arduino code.


sketch_aug08a:48:5: error: 'client' was not declared in this scope

     client.println(payload+"\r");

     ^

exit status 1
'client' was not declared in this scope


So far php there seems to be no error but then again nothing is echoing out.

Sorry i been busy with my dad and his cancers doctors app.


Joseph

**Comment by Therealjosephchrzempiec** [comment]
Hello sorry I'm a little lost on this moded code.

**Comment by Therealjosephchrzempiec** [comment]
Hello let me rephrase this. I got the php code working it's the esp8266 code I'm lost. I'm not sure how to mod the one you have with this new code.

**Comment by Jaunty42** [comment]
Don't worry Soon I'll update with the full source code for arduino.

**Comment by Jaunty42** [comment]
```
#include <WiFi.h>
#include <HTTPClient.h>
#include <Arduino.h>

 
const char* ssid = "YOUR ROUTER SSID";
const char* password =  "YOUR ROUTER PASSWORD";

void setup() {
 
  Serial.begin(115200);
  delay(4000);   //Delay needed before calling the WiFi.begin

  WiFi.begin(ssid, password); 

  while (WiFi.status() != WL_CONNECTED) { //Check for the connection
    delay(1000);
   
    Serial.println("Connecting to WiFi..");
  }
  Serial.println("Connected to the WiFi network");
  }
 
void loop() {

  delay(50);
 
 if(WiFi.status()== WL_CONNECTED){   //Check WiFi connection status
 
int code = 1; // This is the value that you want to store/update

HTTPClient http;

// This is the url where your PHP script is stored.
http.begin("http://yourdomain.com/index.php?updatecode="+code); //Specify the URL

   int httpCode = http.GET();    
   if (httpCode > 0) { //Check for the returning code
    String payload = http.getString();
    Serial.println(httpCode);
  
    
  }
else {
   Serial.println(httpCode);
  Serial.println("Error on HTTP request&quot");
}
http.end(); //Free the resources 

 
 }else{ Serial.println("Error in WiFi connection"); }
 
  //delay(5000);  //Send a request every 10 seconds
 
}

```

**Comment by Jaunty42** [comment]
In above code you have to modify these lines --> 
```

const char* ssid = "YOUR ROUTER SSID";
const char* password =  "YOUR ROUTER PASSWORD";
```
and this -->
```

// This is the url where your PHP script is stored.
http.begin("http://yourdomain.com/index.php?updatecode="+code); //Specify the URL
```

**Comment by Jaunty42** [comment]
Let me know if you face any issue.
